modules/pricer/src/main/java/com/opengamma/strata/pricer/impl/volatility/smile/SabrHaganVolatilityFunctionProvider.java

```diff
@@ -103,13 +108,13 @@ public double volatility(
                   (2 - 3 * rho * rho) / 24)) /
           f1;
     } else {
-      if (DoubleMath.fuzzyEquals(beta, 0, BETA_EPS)) {
+      if (nearZero(beta, BETA_EPS)) {
         double ln = Math.log(forward / k);
         z = nu * Math.sqrt(forward * k) * ln / alpha;
         zOverChi = getZOverChi(rho, z);
         vol = alpha * ln * zOverChi *
             (1 + timeToExpiry * (alpha * alpha / forward / k + nu * nu * (2 - 3 * rho * rho)) / 24) / (forward - k);
-      } else if (DoubleMath.fuzzyEquals(beta, 1, BETA_EPS)) {
+      } else if (nearOne(beta, BETA_EPS)) {
```

Comment:
Do you want comments on more potentially duplicated squares etc in this class or is this simply a review on replacing Math.pow() usages?

Reply:
The goal was not to find all squares and replace them with `pow2()` - I suspect that would be counterproductive. In general, once you go beyond replacing `Math.pow` and the repeated large divisions, you can't tell whether a change makes a difference.